build-service.py

```diff
@@ -178,22 +195,36 @@ def build_request():
 
     # accepts json as string or as part of request json
     try:
-        deploy_conf = reqj["deploy_config"]
-        service_conf = reqj["service_config"]
+        deploy_conf_location = reqj["deploy_config"]
+        service_conf_location = reqj["service_config"]
     except Exception:
         logging.error("malformed config payload")
         return {"err": "Bad request: malformed config payload"}, 400
 
-    try:
-        image_name = build_repo(
-            reqj["repo_url"], reqj["repo_branch"], deploy_conf, service_conf
-        )
-    except Exception as e:
-        logging.error(f"failed to build: {e}")
-        return {"err": f"Failed to build: {e}"}, 500
+    with tempfile.TemporaryDirectory(prefix="kaas-repo-build-") as repo_dir:
+        clone_repo(reqj["repo_url"], reqj["repo_branch"], repo_dir)
+        deploy_conf = get_deploy_conf(deploy_conf_location, repo_dir)
+        service_conf = get_service_conf(service_conf_location, repo_dir)
 
-    logging.info(f"repo built successfully as '{image_name}'")
-    return {"image": image_name}
+        try:
+            image_name = build_repo(
+                repo_dir, reqj["repo_branch"], deploy_conf, service_conf
+            )
+        except Exception as e:
+            logging.error(f"failed to build: {e}")
+            return {"err": f"Failed to build: {e}"}, 500
+
+        # load kube config from ~/.kube/config
+        kubernetes.config.load_kube_config()
+        kubeneretes_api = kubernetes.client.AppsV1Api()
```

Comment:
Please fix globally :)
```suggestion
        kubernetes_api = kubernetes.client.AppsV1Api()
```